src/coreComponents/finiteVolume/TwoPointFluxApproximation.cpp

```diff
@@ -689,10 +689,10 @@ void TwoPointFluxApproximation::addEDFracToFractureStencil( MeshLevel & mesh,
   }
 
   // Add connections EmbeddedSurface to/from CellElements.
-  arrayView1d< localIndex const > const & elemRegionList    = fractureSubRegion.getSurfaceToRegionList();
-  arrayView1d< localIndex const > const & elemSubRegionList = fractureSubRegion.getSurfaceToSubRegionList();
-  arrayView1d< localIndex const > const & elemList          = fractureSubRegion.getSurfaceToCellList();
-  arrayView1d< real64 const >     const & connectivityIndex = fractureSubRegion.getConnectivityIndex();
+  arrayView1d< localIndex const > const elemRegionList    = fractureSubRegion.getSurfaceToRegionList().toViewConst();
```

Comment:
shouldn't this be handled by the getter?

Reply:
So `getSurfaceToRegionList` returns a `arrayView1d< localIndex >` and there's probably a `const` version that returns an `arrayView1d< localIndex const >`. If this is never modified when accessed via the getter then there should only be a `const` version that returns an `arrayView1d< localIndex const >`. However if it does need to be modified via the getter then it should return an `array1d< localIndex > const &`. The benefits of this are

1. If you're doing stuff like in `SurfaceGenerator` where resizing happens then it's much more convenient to hold onto the array instead of getting a view out after resizing.
2. By returning an `array1d< localIndex > const &` you get all the protections of `arrayView1d< localIndex >` (ie no resizing) but you also get the UDC to `arrayView1d< localIndex const >` which as described above is not valid for `arrayView1d< localIndex >` rvalues.

But I didn't want to make this change right now. This should be a pretty transparent change to make though.